includes/AMP/Sanitization.php

```diff
@@ -61,16 +62,25 @@ class Sanitization {
 	 */
 	private $settings;
 
+	/**
+	 * Experiments instance.
+	 *
+	 * @var Experiments Experiments instance.
+	 */
+	private $experiments;
+
 	/**
 	 * Analytics constructor.
 	 *
 	 * @since 1.12.0
 	 *
-	 * @param Settings $settings Settings instance.
+	 * @param Settings    $settings    Settings instance.
+	 * @param Experiments $experiments Experiments instance.
 	 * @return void
 	 */
-	public function __construct( Settings $settings ) {
-		$this->settings = $settings;
+	public function __construct( Settings $settings, Experiments $experiments ) {
```

Comment:
I am surprised this change it not mean updating or creating new tests.

Reply:
We use the injector for tests. New dependencies are automatically added.